app/views/ckey2bibframes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Ckey to Bibframe2 Conversion</h1>
+<div class="btn-group">
+  <%= link_to 'Do another conversion', new_ckey2bibframe_path, class: 'btn btn-md btn-default btn-full' %>
+</div>
+<h3>MarcXML</h3>
+<pre>
+  <%= @ckey2bibframe.marc21_to_xml %>
+</pre>
+<h3>Bibframe2</h3>
+<pre>
+  <%= @ckey2bibframe.marcxml_to_bibframe %>
+</pre>
```

Comment:
Is there any use-case for viewing these files side-by-side?  Basically, wondering whether a table with scroll bars or a couple of text fields with scroll bars might be a good way to view the results?